docs/siem/detections/api/rules-api-create.asciidoc

```diff
@@ -60,145 +65,235 @@ calls, see:
-* `rule`: Every time new signals are detected
+* `rule`: Every time new alerts are detected
 * `1h`: Every hour
 * `1d`: Every day
 * `7d`: Every week
 
-|Yes, when actions are used to send notifications.
+Required when `actions` are used to send notifications.
 
-// |to |String |Time to which data is analyzed each time the rule executes, using a
-// {ref}/common-options.html#date-math[date math range]. For example, `"now-300s"` 
-// means the rule analyzes data until 5 minutes before its starts time. |Yes
+|version |Integer |The rule's version number. Defaults to `1`.
 
-|type |String a|Data type on which the rule is based:
+|==============================================
 
-* `query`: query-based conditions with or without additional filters.
-* `saved_query`: saved search, identified in the `saved_id` field.
-* `machine_learning`: rule based on a {ml} job's anomaly scores.
+===== Optional fields for query and threshold rules
 
-|Yes
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
 
+|exceptions_list |Object[] a|Array of exception containers, which define
+exceptions that prevent the rule from generating alerts even when its other
+criteria are met. The object has these fields:
 
-|threat |<<threats-object-create, threat[]>> |Object containing attack 
-information about the type of threat the rule monitors, see
-{ecs-ref}/ecs-threat.html[ECS threat fields]. |No, defaults to an empty array.
+* `id` (string, required): List ID of the exception container.
+* `namespace_type` (string required): Determines whether the exceptions are
+valid in only the rule's {kib} space (`single`) or in all {kib} spaces
+(`agnostic`). 
+* `type` (string, required): The exception type, which must be either
+a detection rule exception (`detection`) or an endpoint exception (`endpoint`).
 
-|references |String[] |String array containing notes about or references to 
-relevant information about the rule. |No, defaults to an empty array.
+BEN - add link later
+
+|filters |Object[] |The {ref}/query-filter-context.html[query and filter 
+context] array used to define the conditions for when alerts are created from 
+events. Defaults to an empty array.
+
+|index |String[] |Indices on which the rule functions. Defaults to the
+Security Solution indices defined on the {kib} Advanced Settings page
+(*Kibana* → *Stack Management* → *Advanced Settings* →
+`securitySolution:defaultIndex`).
+
+|language |String |Determines the query language, which must be
+`kuery` or `lucene`. Defaults to `kuery`.
 
-|version |Integer |The rule's version number. a|No, defaults to `1`.
+|risk_score_mapping |Object[] a|Overrides generated alerts' `risk_score` with
+a value from the source event:
+
+* `field` (string, required): Source event field used to override the default
+`risk_score`. This field must be an integer.
+* `operator` (string, required): Must be `equals`.
+* `value`(string, required): Must be an empty string (`""`).
```

Comment:
Just a heads up -- I'm fixing some bugs around this area of the code, and will probably refine this structure a bit to clean up the API. It's looking like only `field` will be required, and we'll also add a `risk_score` prop as well (like the severity structure below) for future support of mapping specific values to specific a risk_score. 

This is good for now though, and I'll ping you on that PR with changes.

Reply:
Great - thanks